index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home - ArtScope</title>
+    <link rel="stylesheet" href="styles/styles.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="big-bkg-container">
+<header class="myheader header">
+     <div class="pre-nav">
+        <div class="pre-nav-container">
+           <div class="social-media">
+             <a href="#" class="facebook-icon"><img src="images/facebook-icon.png" alt="Facebook-icon"></a>
+             <a href="#" class="twitter-icon"><img src="images/twitter-icon.png" alt="twitter-icon"></a>
+           </div> 
+
+           <div class="pre-nav-actions">
+             <h5><a>English</a></h5>
+             <h5><a>My Page</a></h5>
+             <h5><a>Logout</a></h5>
+           </div>
+        </div>
+     </div>
+
+    <nav class="nav-bar">
+        
+       <div class="organizer organizer-close">
+         <ul class="menu-sections">
+            <li class="mySpace"><a href="../angelCapstone-project/about.html" class="link-nav">About</a></li>
```

Comment:
- [ ] Why are you directing into `/angelCapstone-project/...`? I can see your `about.html` page is in main directory. Kindly fix this issue.

Reply:
Fixed direction issue. I did it like this the first time when I started doing the project cause they were not linking properly, however, I totally forgot to change that part before submitting, either way, thank you so much for that.